src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { email, validatePassCoincidence } from 'utils/inputValidations';
+
+import wLogo from '../../assets/wLogo.png';
+
+import 'scss/layout.scss';
+import 'scss/components.scss';
+import 'scss/margins.scss';
+import styles from './styles.module.scss';
+
+interface Inputs {
+  name: string;
+  lastName: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+}
+
+function SignUp() {
+  const { register, errors, getValues, setError } = useForm<Inputs>();
+  //   const onSubmit = (data: Inputs) => console.log(data);
+  const onBlur = (): void => {
+    const response = email('correo no válido')(getValues('email'));
+    if (response) {
+      setError('email', {
+        message: response
+      });
+    }
+  };
+
+  return (
+    <div className="row full-height center">
+      <form
+        className={`column full-height center space-around ${styles.formContainer} ${styles.formBackground}`}
+      >
+        <img className={styles.formImage} src={wLogo} alt="wolox-logo" />
+        <div className="column full-width">
+          <label className={`m-bottom-1 ${styles.formLabel}`}>Nombre</label>
+          <input
+            className={styles.formInput}
+            type="text"
+            name="name"
+            ref={register({ required: 'El nombre es requerido', maxLength: '10' })}
+          />
+          {errors.name?.message && <p className={styles.errorMessage}>{errors.name.message}</p>}
+        </div>
+        <div className="column full-width">
+          <label className={`m-bottom-1 ${styles.formLabel}`}>Apellido</label>
+          <input
+            className={styles.formInput}
+            type="text"
+            name="lastName"
+            ref={register({ required: 'El apellido es requerido', maxLength: '20' })}
+          />
+          {errors.lastName?.message && <p className={styles.errorMessage}>{errors.lastName.message}</p>}
+        </div>
+        <div className="column full-width">
+          <label className={`m-bottom-1 ${styles.formLabel}`}>Email</label>
+          <input
```

Comment:
I think is a great idea create a component for inputs where you control errors and labels